docs/modeling-your-data/modeling-your-data-with-dbt/_model-versions.md

```diff
@@ -9,14 +9,27 @@ import {versions} from '@site/src/componentVersions';
 <!-- Note the following tables are wrapped in the additional tags to correctly render the package version, but work the same as normal markdown tables -->
 
 <Tabs groupId="dbt-packages" queryString>
+
+<TabItem value="unified" label="Snowplow Unified" default>
+
+
+<ReactMarkdown children={`
+| snowplow-unified version       | dbt versions        | BigQuery | Databricks | Redshift | Snowflake | Postgres |
+| -------------------------- | ------------------- | :------: | :--------: | :------: | :-------: | :------: |
+| ${versions.dbtSnowplowUnified} | >=1.6.0 to <2.0.0   |    ✅     |     ✅      |    ✅     |     ✅     |    ✅     |
+`} remarkPlugins={[remarkGfm]} />
```

Comment:
So that we can leave the variables uncommented in the project yml file as [this issue ](https://github.com/dbt-labs/dbt-core/issues/6705 )was solved in 1.6.